### DISCUSSION TITLE
Next Release?

### DISCUSSION THREAD

**Discussion by maggo1404** [alicevision/Meshroom#2628] [Q&A]
Hi,

Is a new release version planed?

Best regards maggo1404

**Comment by natowi** [comment]
Yes, there will be a 2025 release :)
There have been many changes and improvements in the back end like moving to QT6 for the UI but also adding many new capabilities. However, to prevent Meshroom binaries from "blowing up" in size, the project needs to be modularized. There are plans for Machine Learning capabilities like masking and feature detection / matching (models and dependencies are big). The main Meshroom installation should stay at a manageable in size, additional features can then be added as needed via a plugin system. This will make Meshroom more versatile.

So an important release is coming up (far more than 300 merged Pull requests in AliceVision/Meshroom) in 2025.

**Reply by LordMaddhi** [reply]
Is there a planned date or quarter for the update?

**Reply by roydenyates** [reply]
![Screenshot from 2025-02-26 16:35:13](https://github.com/user-attachments/assets/b7d3f8bc-416c-498b-9741-141d9b07a8b1)

**Reply by roydenyates** [reply]
Something for which  look forward, with patience  :-)

**Reply by M4ze1234** [reply]
![Screenshot_20250613_083910](https://github.com/user-attachments/assets/181654fc-b6c0-45e8-a170-f57db5f303ae)

**Reply by LordMaddhi** [reply]
i'm already hyped! 😍

**Reply by antoineSIG** [reply]
it's for this weekend or early next week

**Reply by J4ker** [reply]
please move towards semantic versioning

**Reply by MeirArani** [reply]
Has there been any update on this? We're past the halfway point of the year at this point.

**Reply by natowi** [reply]
We are currently in beta testing. So far it looks promising.

https://github.com/alicevision/Meshroom/milestone/16
https://github.com/alicevision/AliceVision/milestone/36

**Reply by natowi** [reply]
The release notes for AliceVision and Meshroom will be really long. So I decided to write up the most interesting news in the [wiki](https://github.com/alicevision/Meshroom/wiki/New-features-in-Meshroom-2025.1). The release is still undergoing testing, but the wait will soon be over.

**Comment by beaki** [comment]
@natowi  thanks for the heads-up, appreciated! 
everybody, for beta testing, just checking out the latest file version as there seems no tag? what do you need? what would help?

**Reply by natowi** [reply]
It is a private beta.

**Comment by natowi** [comment]
Available now :) Have fun! https://github.com/alicevision/Meshroom/discussions/2839
